### PR TITLE
Oauth integration for facebook

### PR DESCRIPTION
**Pull request by kartikjagdale**
1. OAuth Integration for Facebook.
2. Generate JWT Token.
3. Faulty Integration of Instagram oAUth with no user saved as no email id from instagram api
4. Save Longed Lived Facebook Token in db and update it whenever a user logs in to the app.

NOTE: Please Ignore the previous Comments.

New Changes.
Basic Sign in and SIgn up FLow for Twitter.

### CODE REVIEW

**Review by ashish173 — APPROVED** [review]

**Review by chandradot99 — APPROVED** [review]

**Comment by ashish173** [comment]
@kartikjagdale this looks good you can merge it

**Comment by ashish173** [comment]
@kartikjagdale please merge this and if you need any more approvals please ask people in slack.

**Pull request merged**
